### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by Toinane**
**When merged this pull request will:**
- I've updated french language in a lot of files, just missing "modules"

### CODE REVIEW

File: addons/attributes/stringtable.xml [hunk 1]
```diff
@@ -3,94 +3,119 @@
...
         <Key ID="STR_ZEN_Attributes_PlateNumber">
             <English>Plate Number</English>
+            <French>Plaque d'immatriculation</French>
             <Russian>Госномер</Russian>
             <German>Kennzeichen</German>
             <Polish>Tablica rejestracyjna</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_Exec_Tooltip">
             <English>This code will be executed in the unscheduled environment on the machine where this object is local.\nThe object will be passed in the "_this" variable.</English>
+            <French>Ce code sera exécuté dans l'environnement non planifié de la machine où cet objet est local. L'objet sera passé dans la variable "_this".</French>
```
Comment (bux) [inline addons/attributes/stringtable.xml]: misses the line break character `\n`. Could be an issue or not :)

File: addons/common/stringtable.xml [hunk 2]
```diff
@@ -142,6 +153,7 @@
         </Key>
         <Key ID="STR_ZEN_Common_Sides">
             <English>Sides</English>
+            <French>Faces</French>
```
Comment (bux) [inline addons/common/stringtable.xml]: Is this really the correct translation? Not a french myself but "Faces" doesn't sound like the "side" meant by BLUFOR / OPFOR / INDEPENDANT / CIVILIAN but more like the side of e.g. a cube.

File: addons/common/stringtable.xml [hunk 3]
```diff
@@ -160,141 +172,165 @@
...
             <German>Höhe (m)</German>
             <Polish>Wysokość (m)</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Teleport">
             <English>Teleport</English>
+            <French>Téléportation</French>
         </Key>
         <Key ID="STR_ZEN_Common_AutoAddObjects">
             <English>Add Objects to Curators</English>
+            <French>Ajouter les objets aux collections</French>
```
Comment (bux) [inline addons/common/stringtable.xml]: Curators means "Zeus" here.

File: addons/common/stringtable.xml [hunk 4]
```diff
@@ -160,141 +172,165 @@
...
             <Russian>Сдалвть объекты редактируемыми</Russian>
             <German>Objekte zum Kurator hinzufügen</German>
             <Polish>Dodaj obiekty do Zeusa</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_AutoAddObjects_Description">
             <English>Automatically adds spawned objects to all curators.</English>
+            <French>Ajouter automatiquement les objets générés à toutes les collections.</French>
```
Comment (bux) [inline addons/common/stringtable.xml]: as above

File: addons/common/stringtable.xml [hunk 5]
```diff
@@ -160,141 +172,165 @@
...
         <Key ID="STR_ZEN_Common_PreferredArsenal">
             <English>Preferred Arsenal</English>
+            <French>Arsenal préféré</French>
             <Russian>Предпочитаемый арсенал</Russian>
             <German>Bevorzugtes Arsenal</German>
             <Polish>Preferowany arsenał</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_PreferredArsenal_Description">
             <English>Selects the preferred arsenal used by Zeus Enhanced.\nIf ACE Arsenal is not installed, will fall back to BI Virtual Arsenal.</English>
+            <French>Sélectionner l'arsenal préféré utilisé par Zeus Enhanced. Si ACE Arsenal n'est pas installé, il retournera à BI Virtual Arsenal.</French>
```
Comment (bux) [inline addons/common/stringtable.xml]: Misses the line break character

File: addons/doors/stringtable.xml [hunk 6]
```diff
@@ -3,18 +3,21 @@
...
         <Key ID="STR_ZEN_Doors_Configure">
             <English>Configure Doors</English>
+            <French>Configurer les portes</French>
             <Russian>Конфигурация дверей</Russian>
             <German>Türen konfigurieren</German>
             <Polish>Konfiguruj drzwi</Polish>
         </Key>
         <Key ID="STR_ZEN_Doors_NoBuilding">
             <English>Place on a building</English>
+            <French>Placer un bâtiment</French>
```
Comment (bux) [inline addons/doors/stringtable.xml]: "on a building" -> "sur un bâtiment" ?

File: addons/editor/stringtable.xml [hunk 7]
```diff
@@ -3,85 +3,101 @@
...
         <Key ID="STR_ZEN_Editor_UnitRadioMessages_Description">
             <English>Controls whether unit radio messages are played on placement and waypoint orders, waypoints only, or disabled.</English>
+            <French>Contrôle si les messages radio de l'appareil sont diffusés en fonction de l'emplacement et de l'ordre des repères de balisage, des repères de balisage uniquement ou désactivés.</French>
             <German>Steuert, ob die  Einheit eine Funkmeldung absetzt wenn diese platziert wird und eine Anweisung von Wegpunkten erhält oder nur bei Wegpunkten oder niemals.</German>
             <Polish>Kontroluje czy wiadomości radiowe jednostki są odtwarzane przy postawieniu i wyznaczeniu punktu trasy, jedynie przy wyznaczeniu punktu trasy lub czy są wyłączone.</Polish>
             <Russian>Определяет будут ли ИИ подразделения сообщать по радио о своих действиях или нет.</Russian>
         </Key>
         <Key ID="STR_ZEN_Editor_UnitRadioMessages_WaypointsOnly">
             <English>Waypoints Only</English>
+            <French>Points de passage seulement</French>
```
Comment (bux) [inline addons/editor/stringtable.xml]: above you have used "balisage"

**Comment by bux** [comment]
I've added a few comments. 

Disclaimer: not a french myself :)

File: addons/common/stringtable.xml (hunk 2 above)
Comment (Toinane) [reply addons/common/stringtable.xml]: Oh okey! I'll change that!

File: addons/common/stringtable.xml (hunk 3 above)
Comment (Toinane) [reply addons/common/stringtable.xml]: So I should use "Zeus", can you explain what "curators" mean?

File: addons/doors/stringtable.xml (hunk 6 above)
Comment (Toinane) [reply addons/doors/stringtable.xml]: oops! a little omission here

**Comment by Toinane** [comment]
Thanks for the corrections, I'll change it as soon as possible 😄

File: addons/editor/stringtable.xml (hunk 7 above)
Comment (Toinane) [reply addons/editor/stringtable.xml]: yeah, I'll change to "Point de passage", it's more understandable.

File: addons/common/stringtable.xml (hunk 3 above)
Comment (bux) [reply addons/common/stringtable.xml]: Curator is just a fancy word for Zeus. In German the most common use of curator is for describing a role of someone working at a museum. Maybe it's the same word/meaning in french.

Comment (veteran29) [reply addons/common/stringtable.xml]: > Curator is just a fancy word BI is using for Zeus

I'm not sure if he's an arma player. Looking at profile it looks like it's just a translation for Hacktoberfest (sorry if I'm mistaken). Great job anyway.

Comment (Toinane) [reply addons/common/stringtable.xml]: > > Curator is just a fancy word BI is using for Zeus
> 
> I'm not sure if he's an arma player. Looking at profile it looks like it's just a translation for Hacktoberfest (sorry if I'm mistaken). Great job anyway.

You are right, I love translate a lot of technical documentation. This event is a good reason to find a lot of repositories in needs of translation for me. :) But I'm a player too! Just don't know this mod and I will test it when I can! Ahah

Comment (Toinane) [reply addons/common/stringtable.xml]: > Curator is just a fancy word BI is using for Zeus. In German the most common use of curator is for describing a role of someone working at a museum. Maybe it's the same word/meaning in french.

Thank you for the precision!

**Comment by bux** [comment]
@alganthe could you take a look at this?

File: addons/common/stringtable.xml (hunk 3 above)
Comment (bux) [reply addons/common/stringtable.xml]: You're welcome. digitalocean could make a hacktoberfest all year long :)

File: addons/common/stringtable.xml [hunk 8]
```diff
@@ -70,78 +70,91 @@
...
         <Key ID="STR_ZEN_Common_Medium">
             <English>Medium</English>
+            <French>Moyen</French>
             <Russian>Средний</Russian>
             <German>Medium</German>
             <Polish>Średni</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_High">
             <English>High</English>
+            <French>Élevée</French>
```
Comment (alganthe) [inline addons/common/stringtable.xml]: Pas de féminin.

File: addons/common/stringtable.xml [hunk 9]
```diff
@@ -160,141 +173,165 @@
...
         <Key ID="STR_ZEN_Common_AutoAddObjects">
             <English>Add Objects to Curators</English>
+            <French>Ajouter des objets aux curateurs</French>
             <Russian>Сдалвть объекты редактируемыми</Russian>
             <German>Objekte zum Kurator hinzufügen</German>
             <Polish>Dodaj obiekty do Zeusa</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_AutoAddObjects_Description">
             <English>Automatically adds spawned objects to all curators.</English>
+            <French>Ajouter automatiquement ses objets générés à tous les curateurs.</French>
```
Comment (alganthe) [inline addons/common/stringtable.xml]: Pourquoi "ses" ?

File: addons/common/stringtable.xml [hunk 10]
```diff
@@ -160,141 +173,165 @@
...
         <Key ID="STR_ZEN_Common_DisableGearAnim_Description">
             <English>Disables the "Gear" animation from playing when entering Zeus.</English>
+            <French>Désactiver l'animation "Gear" lors de l'entrée dans Zeus.</French>
             <German>Deaktiviert die "Inventar" Animation der Spielerfigur wenn man in dem Zeusmodus geht.</German>
             <Polish>Wyłącza animację grzebania w ekwipunku po wejściu do Zeusa.</Polish>
             <Russian>Отключает анимированную шестерёнку при входе в режим ГМ</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_DarkMode">
             <English>Dark Mode</English>
+            <French>Mode nuit</French>
```
Comment (alganthe) [inline addons/common/stringtable.xml]: Sombre

File: addons/common/stringtable.xml [hunk 11]
```diff
@@ -160,141 +173,165 @@
...
         <Key ID="STR_ZEN_Common_BIVirtualArsenal">
             <English>BI Virtual Arsenal</English>
+            <French>Arsenal BI Virtual</French>
             <Russian>Виртуальный Арсенал</Russian>
             <German>BI Virtuelle Arsenal</German>
             <Polish>Wirtualny arsenał BI</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_AceArsenal">
             <English>ACE Arsenal</English>
+            <French>Arsenal ACE</French>
```
Comment (alganthe) [inline addons/common/stringtable.xml]: Nom propre ?

File: addons/editor/stringtable.xml [hunk 12]
```diff
@@ -3,85 +3,101 @@
...
         <Key ID="STR_ZEN_Editor_DisableLiveSearch_Description">
             <English>Requires pressing Enter or clicking the search button to confirm a search.\nHelps reduce lag when searching through a lot of items.</English>
+            <French>Nécessite d'appuyer sur Entrée ou de cliquer sur le bouton de recherche pour confirmer une recherche. \nAide à réduire le retard dans la recherche d'un grand nombre d'items.</French>
             <German>Suche wird erst ausgeführt, wenn die Eingabetaste oder die Lupe neben dem Eingabefeld betätigt wird.\nHilft, Verzögerungen bei der Suche durch viele Elemente zu reduzieren.</German>
             <Polish>Wymaga wciśnięcia Enter lub kliknięcia przycisku wyszukiwania by potwierdzić wyszukiwanie.\nPomaga zredukować lag przy przeszukiwaniu wielu przedmiotów.</Polish>
             <Russian>Для подтверждения поиска тебуется нажатие клавиши Ввод\nПомогает уменьшить лаг при поиске </Russian>
         </Key>
         <Key ID="STR_ZEN_Editor_DeclutterEmptyTree">
             <English>Declutter Empty Tree</English>
+            <French>Déclencheur arborescence vide</French>
```
Comment (alganthe) [inline addons/editor/stringtable.xml]: Désencombrer, "declutter" signifie de "retirer les éléments non nécessaires"

File: addons/editor/stringtable.xml [hunk 13]
```diff
@@ -3,85 +3,101 @@
...
             <German>Besatzung hinzufügen umschalten</German>
             <Polish>Przełącz 'dołącz załogę'</Polish>
         </Key>
         <Key ID="STR_ZEN_Editor_EjectPassengers">
             <English>Eject Passengers</English>
+            <French>Ejecter les passagers</French>
         </Key>
         <Key ID="STR_ZEN_Editor_EjectPassengers_Description">
             <English>Ejects the passengers of all selected vehicles. If the vehicle is currently flying, the units will be given parachutes.\nUnlike vanilla eject, units will disembark one after another (not immediately, with a short delay in-between).</English>
+            <French>Éjecte les passagers de tous les véhicules sélectionnés. Si le véhicule est actuellement en vol, les unités recevront des parachutes. Contrairement à l'éjection de vanille, les unités débarqueront les unes après les autres (pas immédiatement, avec un court délai entre les deux).</French>
```
Comment (alganthe) [inline addons/editor/stringtable.xml]: >  Contrairement à l'éjection de vanille

Cette phrase est magique, par contre je pense que tu vois l’erreur.

File: addons/common/stringtable.xml (hunk 9 above)
Comment (Toinane) [reply addons/common/stringtable.xml]: J'avais compris au départ que l'on parlait de Zeus, j'avais mal compris, je rectifie

File: addons/common/stringtable.xml (hunk 11 above)
Comment (Toinane) [reply addons/common/stringtable.xml]: Donc comme en Anglais ? Je ne sais pas comment bien gérer celui là

File: addons/editor/stringtable.xml (hunk 12 above)
Comment (Toinane) [reply addons/editor/stringtable.xml]: Donc plutôt "Retirer les éléments non nécessaires de l'arborescence" ?

File: addons/editor/stringtable.xml (hunk 13 above)
Comment (Toinane) [reply addons/editor/stringtable.xml]: Oulah oui ! Je ne sais pas ce que j'ai bu ce soir là, mais c'était puissant ahah !
"Contrairement à l'éjection standard, ..." plutôt !

File: addons/common/stringtable.xml (hunk 11 above)
Comment (alganthe) [reply addons/common/stringtable.xml]: Du moment que toutes les entrées sont similaires ça me va, plus haut vous utilisez "ACE arsenal" et ici "arsenal ACE".

**Comment by mharis001** [comment]
I think this just needs the "ACE Arsenal" vs "Arsenal ACE" changed to be consistent. Other than that looks good to go.

**Pull request merged**